src/useAccessibilityInfo.ts

```diff
@@ -1,43 +1,94 @@
 import {useEffect, useState} from 'react'
-import {
-  AccessibilityInfo,
-  AccessibilityChangeEvent,
-  AccessibilityEvent,
-} from 'react-native'
+import {AccessibilityInfo, AccessibilityChangeEventName} from 'react-native'
 
-export function useAccessibilityInfo() {
-  const [reduceMotionEnabled, setReduceMotionEnabled] = useState(false)
-  const [screenReaderEnabled, setScreenReaderEnabled] = useState(false)
+const SUPPORTS_RN60_ACCESSIBILITY_INFO_API = !!(
```

Comment:
I think that this will be false unless the accessibility features are enabled now?

You probably need to test `!== undefined`

Reply:
The methods listed here were introduced in RN 0.60, prior to that they'd be undefined, resolving `SUPPORTS_RN60_ACCESSIBILITY_INFO_API` to false.

I poked around to see if React Native exposes the version as part of the API but couldn't find anything, if you do know a better way to infer this let me know.

Reply:
Oh, I thought that `AccessibilityInfo.isGrayscaleEnabled` was a boolean, not a function 🤦